.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "bradlc.vscode-tailwindcss",
+    "stylelint.vscode-stylelint",
+    "unifiedjs.vscode-mdx",
+    "dbaeumer.vscode-eslint"
```

Comment:
We also use editorconfig

```suggestion
    "dbaeumer.vscode-eslint",
    "editorconfig.editorconfig"
```

Reply:
Prettier automatically loads EditorConfig (afaik) and VSCode also automatically loads editorconfig, no?